src/modules/dmCommands/commands/notificationping.js

```diff
@@ -11,12 +11,12 @@ export const desc = 'Get pinged when you level up';
  */
 export async function fun(client, message, member) {
     const userDto = userDb.get(member.id);
-    if (!userDto.notifications.get('levelPing')) {
+    if (!userDto.notifications.get('notificationPing')) {
```

Comment:
Note: production database needs to be migrated to transfer over the old `levelPing` values to `notificationPing`

Reply:
i think i'm the only one subscribed right now

Reply:
Is this really nessicary. We can keep the same name behind the scenes no one will know, less code to deal with

Reply:
> i think i'm the only one subscribed right now
Nope you're one of 6